src/fu-remote-list.c

```diff
@@ -236,6 +237,17 @@ fu_remote_list_add_remote(FuRemoteList *self, FwupdRemote *remote)
 	g_ptr_array_add(self->array, g_object_ref(remote));
 }
 
+static gboolean
+fu_remote_list_is_remote_origin_lvfs(FwupdRemote *remote)
+{
+	if (g_str_has_prefix(fwupd_remote_get_id(remote), "lvfs"))
```

Comment:
How about if you look for contains instead of prefix? Or prefix||suffix?

Reply:
Ack, agree, fixed.